kitsune/kbforums/views.py

```diff
@@ -433,20 +433,17 @@ def locale_discussions(request):
         document__locale=request.LANGUAGE_CODE, document__allow_discussion=True
     )
     try:
```

Comment:
Do we need the `try/except` clause here at all?

Reply:
I guess it's in case one of the arguments isn't castable to an int (like `&desc=foobar`)